app/grants/views.py

```diff
@@ -973,7 +973,7 @@ def grants_by_grant_type(request, grant_type):
         if _type.get("keyword") == grant_type:
             grant_label = _type.get("label")
 
-    clr_round, round_start_date, round_end_date, round_active = get_clr_rounds_metadata()
+    _, round_start_date, round_end_date, _ = get_clr_rounds_metadata()
```

Comment:
this conflict with `from django.utils.translation import gettext_lazy as _` why need to be _ ?

Reply:
Yeah good point, we can import that as anything 👍 Theres 124 uses in `app/dashboard/views.py` - should we go for something like `__` - Or do we want to change it to something more meaningful?

Reply:
My opinion is keep the name of what that function return, is more easy to understand and learn about the code.